### DISCUSSION TITLE
Add methods for exponential decay as a framerate-independent alternative to lerp

### DISCUSSION THREAD

**Discussion by passivestar** [godotengine/godot-proposals#9895] [Scripting]
First learned about this when I came across [this article](https://www.rorydriscoll.com/2016/03/07/frame-rate-independent-damping-using-lerp/) and recently Freya reminded us that it's a thing in her [great talk](https://youtu.be/LSNQuFEDOyQ)

I have to confess every time I want framerate-independent interpolation I have to google the formula because i keep forgetting it. This seems to be useful to have in a game engine considering how commonplace interpolation over time is, and my favorite thing about Godot is that it comes with "batteries included" when it comes to useful math functions. It would probably make sense to have this on all classes that have lerp - vectors, basis, color, etc

This is a function for floats from Freya's slide:

<img width="500" alt="Exp" src="https://github.com/godotengine/godot-proposals/assets/60579014/3f9326dd-bac6-4cc4-ba28-fe1c6e67b868"><br>

Could be called `exp_decay`, there's no need for a clever name

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/9803.

**Reply by passivestar** [reply]
well glad to see I'm [not alone](https://github.com/godotengine/godot/pull/92236#issuecomment-2125102189) in not being able to remember the formula 😄

**Reply by SheepCommander** [reply]
YAYYYY

THANK YOU
